tests/api/onboarding-profile.php

```diff
@@ -99,8 +99,10 @@ public function test_schema() {
 		$data       = $response->get_data();
 		$properties = $data['schema']['properties'];
 
-		$this->assertCount( 15, $properties );
+		$this->assertCount( 17, $properties );
```

Comment:
These tests will need to be updated after `skip` is removed.

Reply:
Nice catch @joshuatf! I removed the property and corrected the tests in the commit 1511792